slices/python3.12-venv.yaml

```diff
@@ -0,0 +1,10 @@
+package: python3.12-venv
+
+slices:
+  libs:
+    essential:
+      - python3.12_standard
```

Comment:
Will it suffice with `python3.12_core`? If not, what's missing?

Reply:
I tried with **python3.12_core**, and got below error

```sh
# python3.12 -m venv .venv
/usr/bin/python3.12: No module named venv
```

Then found that venv module is in distribution slice, and added it.
```sh
# python3.12 -m venv .venv
Error: Command '['/.venv/bin/python3.12', '-m', 'ensurepip', '--upgrade', '--default-pip']' returned non-zero exit status 1.
```

Also added internet, net-data slices but still got errors. Then got thinking that most packages on pypi expect to have standard python, but not having it might cause a lot of issues and hours of debugging.